legotrain_helpers.sh

```diff
@@ -0,0 +1,28 @@
+package: legotrain_helpers
+version: "%(tag_basename)s"
+tag: "v1.0.0"
+source: https://github.com/mfasDa/legotrain_helpers
+---
+#!/bin/bash -e
+
+rsync -a --exclude='**/.git' --delete --delete-excluded $SOURCEDIR/ $INSTALLROOT/
+
+# Modulefile
+MODULEDIR="$INSTALLROOT/etc/modulefiles"
```

Comment:
You could use the `alibuild-recipe-tools` to generate the module. See http://alisw.github.io/alibuild/reference.html#runtime-environment for documentation.

Reply:
I was not aware of this - I will take a look. The module is adapted from the aligenmc module, which is also a bash-script-only package.